presto-docs/src/main/sphinx/security/hive.rst

```diff
@@ -0,0 +1,241 @@
+===========================
+Hive Security Configuration
+===========================
+
+The default security configuration of the :doc:`/connector/hive` uses simple
+authentication.  In this mode, the Hive connector connects to a Hadoop cluster
+that has been configured to use ``simple`` authentication. All queries are
+executed as the user who runs the Presto process, regardless of the user who
+submits the query.
+
+The Hive connector provides addional security options to support Hadoop
+clusters that have been configured to use :ref:`Kerberos
+<hive-security-kerberos-support>` for greater security.
+
+When accessing :abbr:`HDFS (Hadoop Distributed File System)`, Presto can
+:ref:`impersonate<hive-security-impersonation>` the end user who is running the
+query. This can be used with HDFS permissions and :abbr:`ACLs (Access Control
+Lists)` to provide additional security for data.
```

Comment:
Is impersonation an additional method, or does it go hand-in-hand with Kerberos support?

Is there anything to mention for the Hive metastore Thrift service in relation to impersonation?